src/components/CardView/CardView.js

```diff
@@ -1,11 +1,17 @@
 import React, { PropTypes } from 'react';
 
 const CardView = (props) => {
-  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+  const { title, text, imageUrl, imageAlt, imageWidth, imageHeight, buttonUrl, buttonText } = props;
 
   let imageEle;
```

Comment:
not sure `ImageEle` sounds nice. Either image or imageElement.
can we make image variable a const and move ternary logic into JSX?
e.g.

`{imageUrl ? imageElement : null }`

to me it explains intention better. What was your thinking behind this approach?